src/logic/operate.js

```diff
@@ -19,8 +22,10 @@ const operate = (numberOne, numberTwo, operator) => {
             res = a.div(b);
```

Comment:
Hi, we need to handle the  `case '÷':`
dividing by zero breaks the app. instead lets set error message like the calculator does.

Reply:
Fixed